docs/test.rst

```diff
@@ -0,0 +1,32 @@
+Testing image
+=============
+
+Concreate is able to run `behave <https://pythonhosted.org/behave/>`_ based test for the image. We suggest you to read behave docs before reading this chapter.
+
+To can test the image by running:
+
+.. code:: bash
+	  $ concreate test
+
+**Test options**
+
+* ``--test-wip`` -- run only test tagged with ``@wip`` tag.
+
+How tests works
+---------------
+Behave test are separate to two parts steps and features. You can place tests in test sub directory next
+to the image descriptor, module descriptor or in a root of a git repository which contains the modules.
```

Comment:
Add information about the `features` and `steps` subdirectories.

Reply:
this is in behave upstream documentation already